### PR TITLE
As/job applications employee apply

### PR DESCRIPTION
**Pull request by Axel1991**

### CODE REVIEW

File: app/views/job_applications/_form.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
-<%= form_with(model: @job_posting, local: true) do |form| %>
-  <% if @job_posting.present? %>
-    <%= f.hidden_field :status, :value => 1 %>
-    <%= f.submit "Approve", class: "btn btn-large btn-success" %>
-  <% end %>
-<% end %>
\ No newline at end of file
+<%# <%= form_with(model: @job_posting, local: true) do |form| %>
+  <%# <% if @job_posting.present? %> %>
+    <%# <%= f.hidden_field :status, :value => 1 %> %>
```
Comment (TiagoSoares) [inline app/views/job_applications/_form.html.erb]: Is it worth deleting the whole file if it is all commented out?

File: app/views/job_applications/new.html.erb [hunk 2]
```diff
@@ -1 +1 @@
-<%= render 'form' %>
\ No newline at end of file
+<%= hello %>
```
Comment (TiagoSoares) [inline app/views/job_applications/new.html.erb]: Development testing code?

File: app/views/job_applications/show.html.erb [hunk 3]
```diff
@@ -1 +1,2 @@
-<h1><%= @job_posting.content %></h1>
\ No newline at end of file
+
+hello
```
Comment (TiagoSoares) [inline app/views/job_applications/show.html.erb]: Development testing code?

File: app/views/job_applications/_form.html.erb (hunk 1 above)
Comment (Axel1991) [reply app/views/job_applications/_form.html.erb]: Removed

File: app/views/job_applications/new.html.erb (hunk 2 above)
Comment (Axel1991) [reply app/views/job_applications/new.html.erb]: Correct - does this need to be removed to merge?

File: app/views/job_applications/show.html.erb (hunk 3 above)
Comment (Axel1991) [reply app/views/job_applications/show.html.erb]: This has been amended.

**Comment by NotUsedBrain** [comment]
please check where fk is for professions and EmployeeProfile, belongs_to is correct

**Pull request merged**